common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -596,7 +596,11 @@ function (VideoPlayer, i18n) {
                     '[Video info]: YouTube returned an error for ' +
                     'video with id "' + self.id + '".'
                 );
-                self.loadHtmlPlayer();
+                // If the video is already loaded in `_waitForYoutubeApi` by the
+                // time we get here, then we shouldn't load it again.
+                if (!self.htmlPlayerLoaded) {
+                    self.loadHtmlPlayer();
+                }
```

Comment:
I probably would have put this check inside `loadHtmlPlayer`, since that's the code that shouldn't run more than once, but this is fine.